### PR TITLE
Add Percona XtraDB Docs

### PR DESCRIPTION
**Pull request by spectro30**
Signed-off-by: Md. Alif Biswas <alif@appscode.com>

### CODE REVIEW

**Comment by spectro30** [comment]
### Before Merge
- Need to update the parent repository
	- Folder name is updated from `percona-xtradb` to `perconaxtradb` (Got rid of the hyphen)
	- Codespan schema check is failing from ProxySQL end

File: docs/guides/percona-xtradb/volume-expansion/volume-expansion/index.md [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+Annotations:  API Version:  ops.kubedb.com/v1alpha1
+Kind:         PerconaXtraDBOpsRequest
+Metadata:
+  UID:               09a119aa-4f2a-4cb4-b620-2aa3a514df11
+Spec:
+  Database Ref:
+    Name:  sample-pxc
+  Type:    VolumeExpansion
+  Volume Expansion:
+    Mariadb:  2Gi
```
Comment (heheh13) [inline docs/guides/percona-xtradb/volume-expansion/volume-expansion/index.md]: percona?

File: docs/guides/percona-xtradb/quickstart/overview/examples/sample-mariadb.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+apiVersion: kubedb.com/v1alpha2
```
Comment (heheh13) [inline docs/guides/percona-xtradb/quickstart/overview/examples/sample-mariadb.yaml]: rename the file name?

File: docs/guides/percona-xtradb/private-registry/_index.md [hunk 3]
```diff
@@ -1,10 +1,10 @@
 ---
-title: Percona XtraDB using Private Registry
+title: Run MySQL using Private Registry
```
Comment (heheh13) [inline docs/guides/percona-xtradb/private-registry/_index.md]: percona xtra db?

**Review by heheh13 — CHANGES_REQUESTED** [review]
- docs/guides/percona-xtradb/autoscaler/compute/overview/images/pxas-compute.png
- docs/guides/percona-xtradb/images/perconaxtradb-lifecycle.png
- https://kubedb-v2-hugo--pr518-add-px-6sjkougz.web.app/docs/v2022.10.18/guides/percona-xtradb/#life-cycle-of-a-perconaxtradb-object

this image contains MariaDB on it 

otherwise looks good to me

File: docs/guides/percona-xtradb/volume-expansion/volume-expansion/index.md (hunk 1 above)
Comment (spectro30) [reply docs/guides/percona-xtradb/volume-expansion/volume-expansion/index.md]: updated

File: docs/guides/percona-xtradb/quickstart/overview/examples/sample-mariadb.yaml (hunk 2 above)
Comment (spectro30) [reply docs/guides/percona-xtradb/quickstart/overview/examples/sample-mariadb.yaml]: fixed

File: docs/guides/percona-xtradb/private-registry/_index.md (hunk 3 above)
Comment (spectro30) [reply docs/guides/percona-xtradb/private-registry/_index.md]: Updated

**Review by heheh13 — APPROVED** [review]

**Pull request merged**